internal/controller/management_controller.go

```diff
@@ -50,11 +51,13 @@ import (
 // ManagementReconciler reconciles a Management object
 type ManagementReconciler struct {
 	client.Client
-	Scheme                 *runtime.Scheme
-	Config                 *rest.Config
-	DynamicClient          *dynamic.DynamicClient
-	SystemNamespace        string
-	CreateAccessManagement bool
+	Manager                            manager.Manager
+	Scheme                             *runtime.Scheme
+	Config                             *rest.Config
+	DynamicClient                      *dynamic.DynamicClient
+	SystemNamespace                    string
+	CreateAccessManagement             bool
+	SveltosDependentControllersStarted bool
```

Comment:
super nit up to you: most of these fields might be actually private and be set in the `setupwithmanager` method

```go
func (r *reconciler) setupwithmanager(mgr) error {
   r.mgr = mgr
   r.scheme = mgr.getscheme()
   r.sveltosStarted = false
   // etc
```